Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj

```diff
@@ -22,8 +22,20 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
     <PackageReference Include="Selenium.WebDriver" Version="3.141.0" />
   </ItemGroup>
+  
+  <!-- Running this using BeforeTagets="Restore" doest work if we run "dotnet restore" from solution folder. 
+       Running it before CollectPackageReferences, will work in both vs & command line. But, this task will be executed in both build & restore multiple times on multi-target project.
+       So, skipping this task if the copied package-lock.json file is same as the source file, to support incremental builds.
+  -->
+  <Target Name="DownloadAxe" Inputs="package-lock.json" Outputs="node_modules\package-lock.json" BeforeTargets="CollectPackageReferences">
+    <Exec Command="npm install" />
```

Comment:
Should this use `npm ci` instead? A developer who actually wants to update `package-lock.json` can always run `npm install` directly

Reply:
i was using npm install, since CollectPackageReferences was being executed multiple times. I am using a fixed version of axe in package.json. now that i have handled this, we can use npm ci.